pyccel/stdlib/external/mpi4py.py

```diff
@@ -1,3 +1,8 @@
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+# pylint: disable=missing-module-docstring
```

Comment:
Please do not disable missing docstrings outside of the tests/ folder

I suggest:
```
Module mapping the mpi4py library functions to the equivalent MPI commands
```